modules/fff/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_FFF)
+
+  set(FFF_DIR ${ZEPHYR_CURRENT_MODULE_DIR})
+
+  zephyr_library()
```

Comment:
fff is AFAICS a header only library.  Then, you don't need any thing to build.  You can remove `zephyr_library()`.

Reply:
You're right, done.